uui-components/src/inputs/DatePicker/CalendarPresets.tsx

```diff
@@ -34,8 +34,8 @@ export class CalendarPresets extends React.Component<CalendarPresetsProps> {
         return (
             <div ref={ this.props.forwardedRef } className={ cx(uuiPresets.container, this.props.cx) } { ...this.props.rawProps }>
                 <div className={ uuiPresets.header }>Presets</div>
-                {this.getPresets().map((item) => (
-                    <div key={ item.name } className={ uuiPresets.item } onClick={ () => this.props.onPresetSet(item) }>
+                {this.getPresets().map((item, index) => (
```

Comment:
Let's use preset object key as a index, it will be always unique. You can add 'key' for each preset in return value in this.getPresets() function:
```
           presets.push({
                    ...this.props.presets[key].getRange(),
                    name: this.props.presets[key].name,
                    key,
                });
```